### ISSUE TITLE
Chat 4.2 Inheritance Issue

### ISSUE THREAD

**Issue by amart241** [cisagov/ScubaGoggles#66]
If 4.1 is non-compliant, then 4.2 is 'disabled' and it can't be selected. Should we change the baseline to address this?

**Comment by LaurenBassett** [comment]
Blocked because waiting on feedback from the TC team. Once it is clear, we can address the setting inheritance. We will revisit after baseline is updated.

**Comment by snarve** [comment]
duplicate of #63, closing this issue
